brave/common/extensions/file_bindings.cc

```diff
@@ -40,10 +40,8 @@ void PostWriteCallback(
 
 FileBindings::FileBindings(extensions::ScriptContext* context)
     : extensions::ObjectBackedNativeHandler(context),
-      worker_pool_(
-          new base::SequencedWorkerPool(3,
-                                        "file_bindings",
-                                        base::TaskPriority::BACKGROUND)) {
+      file_task_runner_(base::CreateSequencedTaskRunnerWithTraits(
```

Comment:
I think this is still not correct. It should block only for the same filename

Reply:
see `JsonPrefStore::GetTaskRunnerForFile`

Reply:
Yeah, you're right, I'll revert to the previous version.